tests/examples/deploy.pp

```diff
@@ -1,7 +1,21 @@
-include jboss
+class { 'jboss':
+  product       => 'jboss-as',
+  version       => '7.1.1.Final',
+  download_url  => 'http://download.jboss.org/jbossas/7.1/jboss-as-7.1.1.Final/jboss-as-7.1.1.Final.zip',
+  enableconsole => true,
+}
 
-jboss::deploy { 'foobar-app':
+$artifact_version  = "1.0.0"
+$artifact_name     = "sample"
+$artifact_fullname = "${artifact_name}-${artifact_version}"
+
+file { "${::jboss::install_dir}/${artifact_name}_deployment.state":
```

Comment:
!resolve Resolve this to simplier version, this cant work either and this is only darkening the example file.

Maybe provide a real, deployable sample application (maybe something on search.maven.org)?